docs/docs/20-usage/20-workflow-syntax.md

```diff
@@ -1,6 +1,12 @@
 # Workflow syntax
 
-The workflow section defines a list of steps to build, test and deploy your code. Steps are executed serially, in the order in which they are defined. If a step returns a non-zero exit code, the workflow and therefore all other workflows and the pipeline immediately aborts and returns a failure status.
+The workflow section defines a list of steps to build, test and deploy your code. Steps are executed serially, in the order in which they are defined.
+
+If a step returns a non-zero exit code, the workflow and therefore the whole pipeline immediately aborts and returns a failure status.
+
+:::note
+An exception to this rule are steps with a [`status: [failure]`](#status) condition, which will execute them on a failed run.
```

Comment:
```suggestion
An exception to this rule are steps with a [`status: [failure]`](#status) condition, which ensures that they are also executed in the case of a failed run.
```

Reply:
I dislike this change.
Mainly because the `also` implies that `status: [failure]` would make the pipeline execute on failure AND success, when it actually only executes on failure.

Reply:
Changed